docs.feldera.com/docs/tutorials/api-client.md

```diff
@@ -0,0 +1,93 @@
+# REST API client
+
+## Request handling
+
+### Request outcome expectations
+
+The outcome of a request is that it either fails (e.g., DNS lookup failed) without any response
+(no status code nor body), or it succeeds and gets back a response status code and body.
+
+In case of a response, usually it is the Feldera implementation that generated it:
```

Comment:
this phrase is awkward. What is it unusually?
The implementation of what? Maybe replace "implementation" with "service".